_summaries/liu2021swin_1.md

```diff
@@ -1,19 +1,72 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+title: Large-Scale Long-Tailed Recognition in an Open World
+paper: liu2021swin
```

Comment:
I believe the paper tag is liu2019large-scale. Might need to move the contents of this file to liu2019large-scale_1.md

Reply:
also will likely need to rename the images